src/latest_release_note.ts

```diff
@@ -1,28 +1,26 @@
 // TODO: find a better way of loading html into a string
-export const LATEST_RELEASE_NOTE = `<h1>Device Simulator Express Release Notes 🐍📝 (Mar. 10, 2020)</h1>
+export const LATEST_RELEASE_NOTE = `<h1>Device Simulator Express Release Notes 🍰🐍 (Mar. 24, 2020)</h1>
 <p>
-    Thanks for using our extension! We're back with some small updates. <u>Again, please feel free to enable our feature
-        flag in
-        Settings
-        (under the setting titled “<b>deviceSimulatorExpress.previewMode</b>” in the User settings)</u>.
-</p>
+    Thanks for using our extension! <b>Support for micro:bit simulation is now officially released on the
+        Device Simulator Express</b>. 💖✨</p>
 <h2>Changes</h2>
 <p>
-    <h3>Fixes (enabled by default):</h3>
+    <h3>Fixes:</h3>
     <ul>
-        <li>Playing sounds on the CPX simulation now works on MacOS!
-            <ul>
-                <li>Try it out by using <i>cp.play_file(PATH_TO_FILE)</i>.</li>
-            </ul>
-        </li>
-        <li>Previously, the play button on the simulations would sometimes require two clicks. This was fixed.</li>
-        <li>The editor can now support unicode characters (such as Japanese).</li>
+        <li>Fixed serial port selection for serial monitor.</li>
+        <li>Fixed debugger runtime issue that appeared on select machines.</li>
     </ul>
-    <h3>New features (only available with feature flag enabled):</h3>
+    <h3>New features:</h3>
     <ul>
-        <li>BBC micro:bit deploy to device and accompanying serial monitor support.</li>
+        <li>Removed feature flag for the micro:bit simulator!</li>
```

Comment:
This one is a bit tricky. I think it's good to point it out, since some users might remember the checkbox from previous releases, but also I am not quite sure categorizing it as a new feature is accurate.
What do you think?

Reply:
So something like fixes, changes and features?